### PR TITLE
[frameworks][now-static-build] Fix Docusaurus build command

### PR DESCRIPTION
**Pull request by AndyBitz**
Fix Docusaurus build command.

### CODE REVIEW

File: packages/now-static-build/src/frameworks.ts [hunk 1]
```diff
@@ -82,7 +82,7 @@ const frameworkList: Framework[] = [
     name: 'Docusaurus',
     slug: 'docusaurus',
     dependency: '@docusaurus/core',
-    buildCommand: 'docusaurus-build',
+    buildCommand: 'docusaurus build',
```
Comment (styfle) [inline packages/now-static-build/src/frameworks.ts:85]: We need to differentiate docusaurus 1 and docusaurus 2.

File: packages/frameworks/frameworks.json [hunk 2]
```diff
@@ -133,10 +133,10 @@
     },
     "settings": {
       "buildCommand": {
-        "placeholder": "`npm run build` or `docusaurus-build`"
+        "placeholder": "`npm run build` or `docusaurus build`"
       },
       "devCommand": {
-        "value": "docusaurus-start --port $PORT"
+        "value": "docusaurus start --port $PORT"
```
Comment (styfle) [inline packages/frameworks/frameworks.json:139]: I think we need an additional entry here to separate 1.x and 2.x

Can you change `matchContent` to look for 1 or 2?

Comment (AndyBitz) [reply packages/frameworks/frameworks.json:139]: It already does that

File: packages/now-static-build/src/frameworks.ts (hunk 1 above)
Comment (AndyBitz) [reply packages/now-static-build/src/frameworks.ts:85]: We only support Docusaurus 2 for detection here

**Review by styfle — CHANGES_REQUESTED** [review]
Upon further investigation, I don't think this is a great solution.

Previously we detected `docusaurus@1` and this changes to `docusaurus@2` but it will break for `docusaurus@1`.

Let's add detection for both versions since `docusaurus@2` is still in alpha stages (the latest version is `2.0.0-alpha.40`).

**Review by styfle — APPROVED** [review]
I see now that we have a separate "Docusaurus 1.0" entry.

This PR is fixing the Docusaurus 2 entry, thanks!

**Pull request merged**